### ISSUE TITLE
Access to external drives

### ISSUE THREAD

**Issue by Exalm** [debarshiray/toolbox#3]
Currently it seems it's impossible to access external drives from inside the container.

**Comment by tosmi** [comment]
i just modified my local toolbox fork to include `/run/media/${USER}` in the volumes list. this seems to work. 

@debarshiray: any change a pull request for this would be merged (i would open one...)

thanks
toni

**Comment by tosmi** [comment]
hm, i just discovered that on a freshly booted host there is no /run/media/$USER folder. seems like the folder gets created when you run `udisksctl`. you could still map /run/media into the container, but you _first_ have to mount your external drive and _then_ start the container. other wise the permissions are not correct. 

maybe there is an easier way?

**Comment by debarshiray** [comment]
The proper long-term solution in my opinion would be to teach `udisks` to create `/run/media` during boot with `systemd-tmpfiles`. I submitted https://github.com/storaged-project/udisks/pull/641 for that. For the time being, we can carry the snippet ourselves.

Note that this isn't an issue on Silverblue where `/usr/lib/tmpfiles.d/rpm-ostree-0-integration.conf` already takes care of creating `/run/media`, but it would be nice to remove this delta between Silverblue and legacy Fedora Workstation by having `udisks` take care of this.

As for which locations to bind mount, it seems easiest to just mount the top-levels with the `rslave` option. The `rslave` option is important because otherwise paths that get mounted inside the top-levels won't be accessible from the bind mount.

I pushed a rough patch in #78 but it's missing the build system voodoo to install the `tmpfiles.d` snippet.
